etl/steps/data/explorers/wid/2023-01-27/world_inequality_database.py

```diff
@@ -0,0 +1,34 @@
+"""World Inequality Database explorer data step.
+
+Loads the latest WID data from garden and stores a table (as a csv file).
+(Using the structure of the EMDAT Natural Disasters step)
+
+"""
+
+from copy import deepcopy
+
+from owid.catalog import Dataset
+
+from etl.helpers import PathFinder
+from etl.paths import DATA_DIR
+
+N = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+
+    # Load the latest dataset from garden.
+    dataset_garden_latest_dir = sorted((DATA_DIR / "garden" / "wid").glob("*/world_inequality_database"))[-1]
+    ds_garden = Dataset(dataset_garden_latest_dir)
+
+    # Load tables with yearly and decadal data.
+    tb_garden = ds_garden["world_inequality_database"]
+
+    # Initialize a new grapher dataset and add dataset metadata.
+    ds_explorer = Dataset.create_empty(dest_dir)
```

Comment:
Here I think `create_dataset` will not work, since `formats` cannot be specified, so keep it as it is now. I'll create an issue.

Reply:
Actually, I've just created [another PR](https://github.com/owid/etl/pull/826) that should fix the issue. So, once it's merged, you can git pull changes from master to your current branch, and then reduce the code to simply:
```
ds_explorer = create_dataset(dest_dir, tables=[tb_garden], default_metadata=ds_garden.metadata, formats=["csv"])
ds_explorer.save()
```
(and don't forget to import `create_dataset`). But if this is too much hassle, you can ignore it.